go.mod

```diff
@@ -1,16 +1,16 @@
 module gorm.io/driver/postgres
 
-go 1.19
+go 1.21
```

Comment:
Hi, please avoid making this change to ensure compatibility with projects that are still running on older versions of Go.

Reply:
Using a custom TimestampCodec with ScanLocation set to the parsed time zone is only supported starting from pgx version 5.6.0, which requires Go 1.20.0. Since the go.mod file in this PR currently specifies Go 1.19, the Go version must be upgraded to at least 1.20.0. Without this change, this PR will fail to compile after merging.